SDL_GameControllerGetStringForAxis.mediawiki

```diff
@@ -28,6 +27,50 @@ Returns a string for the given axis, or NULL if an invalid axis is
 specified. The string returned is of the format used by
 [[SDL_GameController]] mapping strings.
 
+== Code Examples == 
+<!-- #Example1 Start -->
+<syntaxhighlight lang='c++'>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <SDL.h>
+
+int main(int argc, char* argv[])
+{
+    if(SDL_Init(SDL_INIT_GAMECONTROLLER) < 0)
+    {
+        SDL_LogError(SDL_LOG_CATEGORY_APLLICATION, "Error while initializing SDL2 library : %s\n", SDL_GetError());
+        return EXIT_FAILURE;
+    }
+
+    SDL_Event event;
```

Comment:
SDL uses C89, so variable declarations need to go at the top of the block.

I appreciate you slogging through these requirements, I know it's not your comfortable style.

Thanks!

Reply:
No problem. I'll do as needed. Worry not.